scripts/nqp/actions/skills.py

```diff
@@ -129,17 +129,20 @@ def apply(self) -> Iterator[Tuple[EntityID, List[Effect]]]:
         """
         An iterator over pairs of (affected entity, [effects])
         """
-        entity_names = []
+        applied_entities: collections.defaultdict = collections.defaultdict(lambda: 1)
```

Comment:
collections.defaultdict is like a normal dict but has a specified default value for every key.